sn_node_manager/src/add_services/mod.rs

```diff
@@ -78,6 +78,11 @@ pub async fn add_node(
         check_port_availability(port_option, &node_registry.nodes)?;
     }
 
+    let owner = match &options.owner {
+        Some(owner) => Some(owner.to_lowercase()),
```

Comment:
shall we raise a warning here in case an `UpperCase` input detected ?

at least a `warning log` ?

Reply:
Good idea.